Pachanga/SessionView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  SessionView.swift
+//  Pachanga
+//
+//  Created by Nacho Alaves on 3/8/23.
+//
+
+import SwiftUI
+
+struct SessionView: View {
+    @State private var enoughMembers = false
+    @State private var enoughMaterials = false
+    
+    var mySession = Session(participants: Player.samplePlayers, date: Date.now, location: "El Campello")
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Section("Basic info") {
+                    HStack {
+                        Text("Date")
+                        Spacer()
+                        Text(mySession.date, format: .dateTime.day().month())
+                            .foregroundColor(.secondary)
+                    }
+                    HStack {
+                        Text("Time")
+                        Spacer()
+                        Text(mySession.date, format: .dateTime.hour().minute())
+                            .foregroundColor(.secondary)
+                    }
+                    HStack {
+                        Text("Location")
+                        Spacer()
+                        Text(mySession.location)
+                            .foregroundColor(.secondary)
+                    }
+                }
+                
+                Section("Players") {
+                    HStack {
+                        Text("Javier Alaves")
+                        Spacer()
+                        Image(systemName: "volleyball.fill")
+                    }
+                    HStack {
+                        Text("Nacho Alaves")
+                        Spacer()
+                        Image("net")
+                            .resizable()
+                            .frame(width: 25, height: 25)
+                    }
+                    HStack {
+                        Text("Antonio Marcos")
+                    }
+                    HStack {
+                        Text("Adam Abram")
+                    }
+                }
+                
+                Section {
+                    HStack {
+                        Spacer()
+                        VStack{
+                            Button("Finish Session"){
```

Comment:
Not sure there's value in this action really at this point, would remove for now